README.md

```diff
@@ -7,66 +7,122 @@
-| React             | `@rdfrontier/plugin-react`        | Pre-Technical Proposal |
-| QA                | `@rdfrontier/plugin-qa`           | Ready                  |
-| Mobile            | `@rdfrontier/plugin-mobile`       | Pre-Technical Proposal |
-| React Native      | `@rdfrontier/plugin-react-native` | Technical Proposal     |
-| Infrastructure    | `@rdfrontier/plugin-iac`          | Pre-Technical Proposal |
+| Name | Package | Status | More Information
+| --- | --- | --- | --- |
+| Backend           | `@rdfrontier/plugin-backend`      | Pre-Technical Proposal | [ Read More ](https://github.com/realdecoy/frontier) |
+| Spring Boot       | `@rdfrontier/plugin-java`         | Pre-Technical Proposal | [ Read More ](https://github.com/realdecoy/frontier) |
+| Javascript        | `@rdfrontier/plugin-js`           | Pre-Technical Proposal | [ Read More ](https://github.com/realdecoy/frontier) |
+| .Net              | `@rdfrontier/plugin-dotnet`       | Pre-Technical Proposal | [ Read More ](https://github.com/realdecoy/frontier) |
+| Frontend          | `@rdfrontier/plugin-frontend`     | Pre-Technical Proposal | [ Read More ](https://github.com/realdecoy/frontier) |
+| Vue               | `@rdfrontier/plugin-vue`          | Ready                | [ Read More ](https://github.com/realdecoy/frontier/tree/main/packages/frontier-plugins/plugin-vue) |
+| React             | `@rdfrontier/plugin-react`        | Pre-Technical Proposal | [ Read More ](https://github.com/realdecoy/frontier) |
+| QA                | `@rdfrontier/plugin-qa`           | Ready                  | [ Read More ](https://github.com/realdecoy/frontier/tree/main/packages/frontier-plugins/plugin-qa) |
+| Mobile            | `@rdfrontier/plugin-mobile`       | Pre-Technical Proposal | [ Read More ](https://github.com/realdecoy/frontier) |
+| React Native      | `@rdfrontier/plugin-react-native` | Technical Proposal     | [ Read More ](https://github.com/realdecoy/frontier) |
+| Infrastructure    | `@rdfrontier/plugin-iac`          | Pre-Technical Proposal | [ Read More ](https://github.com/realdecoy/frontier) |
 
 &nbsp;
 &nbsp;
 
 ### Libraries
 
-| Name | Package | Status | 
-| --- | --- | --- |
-| Frontier Standard Library  | `@rdfrontier/typekit`      | Ready |
-| Frontier TypeKit           | `@rdfrontier/stdlib`       | Ready |
+| Name | Package | Status | More Information
+| --- | --- | --- | --- |
+| Frontier Standard Library  | `@rdfrontier/stdlib`      | Ready | [ Read More ](https://github.com/realdecoy/frontier/tree/main/packages/frontier-libraries/stdlib) |
+| Frontier TypeKit           | `@rdfrontier/typekit`       | Ready | [ Read More ](https://github.com/realdecoy/frontier/tree/main/packages/frontier-libraries/typekit) |
+
+&nbsp; &nbsp; &nbsp;
+
+
 
 &nbsp;
+&nbsp; 
 &nbsp;
-&nbsp;
 
 
+### Development Instructions 
+
+#### Development
+
+```bash
+npm install
+npm link
+```
+
+&nbsp; &nbsp; &nbsp;
+
+#### Testing
+
+##### Run Unit Test Suite
```

Comment:
This inst a sufficient enough description, because the test suite its complex, we need more data